src/main/groovy/org/scoverage/ScoverageExtension.groovy

```diff
@@ -146,10 +150,23 @@ class ScoverageExtension {
                 if (extension.highlighting) {
                     parameters.add('-Yrangepos')
                 }
+                if (extension.encoding) {
```

Comment:
This part was not actually necessary in my build, but I suspect that this is only because I was using the Gradle daemon, where as in the case of not using it, it may be that the other settings (in the fork options) will be irrelevant and this one will actually matter.